### ISSUE TITLE
[BUG] Cast error during loading network video

### ISSUE THREAD

**Issue by kkdlau** [jhomlala/betterplayer#187]
**Describe the bug**
I tried to load video using the example code:
```dart
BetterPlayer.network(
    'https://commondatastorage.googleapis.com/gtv-videos-bucket/sample/ForBiggerBlazes.mp4')
```
However, it led to runtime error:
`type 'double' is not a subtype of type 'int' in type cast` on line 202

**To Reproduce**
Build the widget above and run, a cast error will be produced.

**Flutter doctor**
```
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel stable, 1.22.5, on Mac OS X 10.15.4 19E287 darwin-x64, locale en-HK)
 
[✓] Android toolchain - develop for Android devices (Android SDK version 29.0.1)
[✓] Xcode - develop for iOS and macOS (Xcode 12.3)
[✓] Android Studio (version 3.4)
[✓] VS Code (version 1.52.1)
 
[✓] Connected device (1 available)   
```

**Better Player version**
- Version: `0.0.41`

**Smartphone (please complete the following information):**
 - Device: iPhone XR
 - OS: iOS 14.1

**Comment by jbworldllc** [comment]
@kkdlau I had gotten same error .. Found a temporary work around..

Got it to work after editing code on
Page : method_channel_media_dart.dart
line : 202
Change Line TO : **size: Size(map['width'] ?? 0.0, map['height'] ?? 0.0),**

**Comment by jhomlala** [comment]
Fixed in 0.0.42.
